install-dev/upgrade/upgrade.php

```diff
@@ -45,8 +45,11 @@
     define('_PS_ADMIN_DIR_', base64_decode($_GET['adminDir']));
 }
 
-require_once dirname(__FILE__).'/../init.php';
+require_once __DIR__ . '/../../vendor/autoload.php';
```

Comment:
should you check for its existence like in the module?

Reply:
I don't think so, because without that autoloader PrestaShop won't work.

Reply:
Unless you forgot to run `composer install` But then nothing would work, so yeah you're right